src/metabase/driver/crate.clj

```diff
@@ -55,6 +57,39 @@
 (defn- string-length-fn [field-key]
   (hsql/call :char_length field-key))
 
+(defn- describe-table-fields
+  [database, driver, {:keys [schema name]}]
+  (let [columns (jdbc/query
+                  (sql/db->jdbc-connection-spec database)
+                    [(format "select column_name, data_type as type_name from information_schema.columns
+                              where table_name like '%s' and table_schema like '%s'" name schema)])]
+    (set (for [{:keys [column_name type_name]} columns]
+      (merge {:name      column_name
+              :custom    {:column-type type_name}
+              :base-type (or (column->base-type (keyword type_name))
+                           (do (log/warn (format "Don't know how to map column type '%s' to a Field base_type, falling back to :type/*." type_name))
+                             :type/*))})))))
```

Comment:
to clarify, the bodies of things like `let`, `for`, or `doseq` (e.g., forms with bodies) should be indented two spaces. But for other things like `do` and normal functions the arguments should be aligned. For functions with arguments on following lines (`jdbc/query` in the example above) you align args with the function name.

At any rate there's no case where you'd indent things like you did here

```clojure
(jdbc/query
  (sql/db->jdbc-connection-spec database)
    [(format "select column_name, data_type as type_name from information_schema.columns
              where table_name like '%s' and table_schema like '%s'" name schema)])
```

that's confusing since the second arg is indented more than the first. 

Take a minute to read [the section about indentation in the Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide#source-code-layout--organization)